packages/venia-ui/lib/index.css

```diff
@@ -2,6 +2,8 @@
 
 :global(*) {
     box-sizing: border-box;
+    margin: 0;
+    padding: 0;
```

Comment:
I used this shortcut on the styleguide app, but this seems like something that could cause widespread or hard-to-find regressions. Does it seem otherwise to you?

Reply:
This was to fix an issue with radio buttons positioning. This looked fine in multiple browsers. However, to be safe, I will move this reset into the RadioGroup component.